add_spec_to_comps

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+shopt -s nullglob
+
+SPEC=$1
+
+if [[ -z $SPEC ]] ; then
+	echo "Usage: $0 SPEC_FILE" >&2
+	exit 1
+fi
+
+SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
+PACKAGE_SUBDIR=$(basename "$(dirname "$(dirname "$SPEC")")")
+
+case $PACKAGE_SUBDIR in
+	client)
+		COMPS_PREFIX=foreman-client-
+		;;
+	foreman)
+		COMPS_PREFIX=foreman-el
+		;;
+	katello)
+		COMPS_PREFIX=katello-
+		;;
+	plugins)
+		COMPS_PREFIX=foreman-plugins-
+		;;
+	*)
+		COMPS_PREFIX=
+		;;
+esac
+
+if [[ -z $COMPS_PREFIX ]] ; then
+	echo "Unable to determine package directory for '$SPEC'" >&2
```

Comment:
I am running into this error message testing it:

```
 ./add_gem_package.sh bootsnap default foreman
~/workspace/upstream/packaging/foreman-packaging/packages/foreman/rubygem-bootsnap ~/workspace/upstream/packaging/foreman-packaging
add bootsnap-1.18.4.gem 
ok                                
(recording state in git...)
~/workspace/upstream/packaging/foreman-packaging
Unable to determine package directory for 'rubygem-bootsnap.spec'
Guessed '.'
```

Reply:
I just pushed without addressing this, but having it pushed makes it safer to not lose work.

Reply:
Fixed by passing in the absolute path instead of just the filename. Looks like in some files we have `SPEC_FILE` as a relative value while in others it's the absolute path.